force-app/main/default/lwc/rd2ChangeEntryValue/rd2ChangeEntryValue.js

```diff
@@ -0,0 +1,34 @@
+import { LightningElement, api } from 'lwc';
+import { NavigationMixin } from 'lightning/navigation';
+
+export default class Rd2ChangeEntryValue extends NavigationMixin(LightningElement) {
+    @api value;
+    @api idValue;
```

Comment:
Could we rename this variable to make it more clear what it's used for since it's a public property? Something like entryRecordId?

Reply:
How about `recordIdValue` since entry is already in the name of the component?